### PR TITLE
EX-3450 Create Sovrn docs on how to use prebid video

### PR DESCRIPTION
**Pull request by mikhalovich**

### CODE REVIEW

**Review by jrosendahl — APPROVED** [review]
need to wait for prebid pr to be ready

**Review by cpabst — COMMENTED** [review]
I need to talk to John R. before I approve this. It seems to say we will support video in situations I'm pretty sure we don't.

File: dev-docs/bidders/sovrn.md [hunk 1]
```diff
@@ -20,3 +20,175 @@ gvl_id: 13
...
+| `api`            | optional | List of supported API frameworks for this impression. Refer to [API Frameworks](api-frameworks)                                | `[1, 2, 3]`     | `integer array` |
+ 
+
+### Note
+
+[Protocols list](https://docs.prebid.org/dev-docs/bidders/conversant.html#protocols)
+
+#### MIME types:
+
+##### iOS with VPAID2:
```
Comment (cpabst) [inline dev-docs/bidders/sovrn.md]: I'm thinking we should not list iOS here, as we do not support InApp yet.

**Review by cpabst — COMMENTED** [review]
Will talk to John R again tomorrow.

**Comment by cpabst** [comment]
Changes have been merged

**Pull request closed without merging**